test/unit/std/charconv_int_test.cpp

```diff
@@ -231,14 +233,19 @@ TYPED_TEST(integral_from_char_test, hexadicimal_number)
 
 TYPED_TEST(integral_from_char_test, to_chars)
 {
-    TypeParam val{120};
-    std::array<char, 10> buffer{};
+    uint8_t max_num_digits = static_cast<uint8_t>(std::log10(std::numeric_limits<TypeParam>::max())) + 1;
+    std::array<char, 20> buffer{};
 
-    auto res = std::to_chars(buffer.data(), buffer.data() + buffer.size(), val);
+    for (uint8_t num_digits = 1; num_digits <= max_num_digits; ++num_digits)
+    {
+        TypeParam val = seqan3::pow(10, num_digits - 1);
```

Comment:
```suggestion
        // 1, 10, 100, 1000, 10000, ..., 10^(ceil_log_10(max_number))
        TypeParam val = seqan3::pow(10, num_digits - 1);
```

Reply:
I would change it to 1, 12, 123, 1234, 12345, 123456, ..., 123456789,  1234567890, ...

```c++
TypeParam value = 1;
value *= 10 + num_digits % 10;
```

Reply:
done